src/components/inputs/Selector.tsx

```diff
@@ -82,8 +85,6 @@ export default function Selector({
   const open = Boolean(anchorEl)
   const id = open ? 'autocomplete-popper' : ''
 
-  const [inputStatusValue, setInputStatusValue] = useState('')
```

Comment:
Is there any particular reason for us to remove this from the component? I think it is better if we maintain this state internally in the component scope.

Reply:
The `FilterBar.tsx` component requires us to track the value of selector text value to see where it resets back to an empty string. This was extracted out because hooks like `useState` cannot be called called conditionally so we can't just have that for the FilterBar only.

I tried to use a ref before this to extract the text value but that was lagging a render behind as well.